### DISCUSSION TITLE
Any suggestions on getting Push notifications to work on OpenFire to Siskin.IM

### DISCUSSION THREAD

**Discussion by MastUniv** [tigase/siskin-im#195] [Q&A]
Hey folks,

i'm relatively new to XMPP chat and i've setup a OpenFire Server 4.7.1 but i can't seem to get Push Notifications to be received by my Siskin.IM 7.3 app. I've also got the Android app, Blabber.IM 3.1.1.and that does seem to be getting Push Notifications so i'm not sure if it's missing something on the configuration side that Blabber.Im doesn't need.

I have push notifications turned on in the settings but the push notifications is greyed out in the Group Chat. Push Notifications aren't being received for either P2P or Group chats.

Cheers! I look forward to working with you to resolve this.

**Comment by woj-tek** [comment]
Do you receive 1-to-1 notifications or those are missing as well? Please make sure your server can establish s2s connection to push.tigase.im

Group Chat notifications requires Tigase XMPP Server or Prosody to work (it's currently a custom extension)
